trial-config-service/src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
how are these files generated?

Reply:
These files are currently edited when we add edit the DB schema. We should start revisioning them and then maybe start using Liquibases generate functionality - that said Im not sure how that works in a container/microservice environement?